examples/wearable/UIComponents/contents/navcontrols/indexscrollbar/indexScrollbar.js

```diff
@@ -46,17 +47,43 @@
 			});
 		}
 
+		listview = tau.engine.getBinding(listviewElement);
+
 		// Add IndexScrollbar index "select" event handler.
 		indexScrollbarElement.addEventListener("select", function (ev) {
 			var divider,
-				idx = ev.detail.index;
+				idx = ev.detail.index.trim(),
+				index;
```

Comment:
index = -1 ?

Reply:
We don't have to initialize this value twice.